packages/suite/src/components/suite/WebUsbButton.tsx

```diff
@@ -1,16 +1,23 @@
 import TrezorConnect from '@trezor/connect';
-import { Button, ButtonProps } from '@trezor/components';
-import { Translation } from 'src/components/suite';
+import { ButtonProps, IconButton } from '@trezor/components';
+import { useIntl } from 'react-intl';
+import messages from 'src/support/messages';
 
-export const WebUsbButton = (props: Omit<ButtonProps, 'children'>) => (
-    <Button
-        {...props}
-        icon="SEARCH"
-        onClick={e => {
-            e.stopPropagation();
-            TrezorConnect.requestWebUSBDevice();
-        }}
-    >
-        <Translation id="TR_CHECK_FOR_DEVICES" />
-    </Button>
-);
+export const WebUsbButton = (props: Omit<ButtonProps, 'children'>) => {
+    const intl = useIntl();
+    return (
+        <div data-test="TR_CHECK_FOR_DEVICES">
+            <IconButton
+                {...props}
+                icon="REFRESH"
```

Comment:
New icon

```
<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16" fill="none">
  <path d="M13 5.5L14.5 4L13 2.5" stroke="#333333" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
  <path d="M2.5 8C2.50165 6.93964 2.92361 5.92319 3.6734 5.1734C4.42319 4.42361 5.43964 4.00165 6.5 4H14.5" stroke="#333333" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
  <path d="M4 10.5L2.5 12L4 13.5" stroke="#333333" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
  <path d="M14.5 8C14.4983 9.06036 14.0764 10.0768 13.3266 10.8266C12.5768 11.5764 11.5604 11.9983 10.5 12H2.5" stroke="#333333" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
</svg>
```

Reply:
just FYI we can't use svgs from Figma 1:1. Usually I open file and edit it a bit:
- delete `width, height , stroke, fill`
- optionally add `viewBox` if needed
- convert strokes to fill (e.g.: https://iconly.io/tools/svg-convert-stroke-to-fill)
- add `fill-rule="evenodd"`

For example this is the result:
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" fill-rule="evenodd"><path d="M12.824 1.742c-.337.125-.611.467-.61.758.001.175.071.313.278.551l.177.202H9.514c-3.156 0-3.514.009-3.937.096a4.75 4.75 0 0 0-3.334 2.566 4.61 4.61 0 0 0-.475 1.862c-.017.386.008.547.11.703.201.308.864.359 1.161.091.14-.126.185-.274.214-.695.054-.787.312-1.439.786-1.989.529-.612 1.252-1 2.076-1.112.138-.019 1.284-.028 3.381-.028h3.173l-.177.202c-.098.112-.2.253-.227.314a.6.6 0 0 0 .007.491c.072.154.291.379.45.462a.639.639 0 0 0 .558-.001c.075-.038.407-.343.884-.812.42-.412.812-.789.871-.837.132-.109.245-.369.245-.566 0-.197-.113-.457-.245-.566a35.475 35.475 0 0 1-.871-.837c-.472-.464-.808-.773-.879-.808-.146-.074-.337-.094-.461-.047m1.456 5.529c-.207.046-.364.159-.441.317-.043.089-.065.219-.094.545-.041.472-.102.748-.24 1.09a3.03 3.03 0 0 1-.775 1.124c-.446.428-.932.687-1.583.843-.186.044-.438.048-3.518.056l-3.317.009.173-.183c.398-.422.399-.756.004-1.125-.187-.176-.375-.256-.545-.233-.229.031-.336.115-1.12.885-.42.411-.81.79-.868.842-.113.101-.249.408-.249.559 0 .151.136.458.249.559.058.052.448.431.868.842.784.77.891.854 1.12.885.17.023.358-.057.545-.233.395-.369.394-.703-.002-1.123l-.171-.181 3.355-.009c3.616-.01 3.455-.004 4.022-.15 2.038-.524 3.513-2.39 3.541-4.483a1.831 1.831 0 0 0-.031-.427.622.622 0 0 0-.401-.384 1.382 1.382 0 0 0-.522-.025" /></svg>
```

Otherwise it doesn't fill icon properly.